Makefile.MinGW

```diff
@@ -1,7 +1,11 @@
+$(shell windres MSWindows\icon.rc -O coff -o icon.res)
+$(shell windres MSWindows\exe_header.rc -O coff -o exe_header.res)
 
-CFLAGS=-I/include/ -Wall --std=c99
```

Comment:
Normally MinGW maps windows paths to unix-like ones. Using native Windows paths shouldn't be necessary ...